modules/navigation.js

```diff
@@ -0,0 +1,20 @@
+function showContent(section) {
+  const contentSections = document.querySelectorAll('.content');
+  contentSections.forEach((content) => {
+    content.classList.remove('active');
+  });
+  section.classList.add('active');
+}
+
+/* eslint-disable import/prefer-default-export */
+
+export function initializeNavigation() {
```

Comment:
- [ ] Save here too as pointed out earlier :point_up: Kindly change this function to **arrow** function syntax.